bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties

```diff
@@ -33,6 +33,13 @@ thing-type.luxtronikheatpump.heatpump.channel.channel256.label = Kanal 256
 thing-type.luxtronikheatpump.heatpump.channel.channel258.label = Kanal 258
 thing-type.luxtronikheatpump.heatpump.channel.channel259.label = Kanal 259
 thing-type.luxtronikheatpump.heatpump.channel.channel260.label = Kanal 260
+thing-type.luxtronikheatpump.heatpump.channel.channel261.label = Kanal 261
```

Comment:
I guess changes to that file shouldn't be made, as translations are handled by an external tool.